example_input_files/loop.input

```diff
@@ -1,32 +1,51 @@
 # *******************************************************
-# Input parameters for cvc-code
-# Filters the existing loops in 1;) loop types
-#                               2;) Momentum transfer
-# Optionally perform spin projection as well
-# for loop_extract executable
+# * input parameters for cvc-code
 # *******************************************************
-T =  64
-LX = 32
-LY = 32
-LZ = 32
-mu = 0.0030
-kappa = 0.1400645
-Nconf = 1240
-csw = 1.740000
+
+T =  96
+LX = 48
+LY = 48
+LZ = 48
+mu = 0.0025
+kappa = 0.1394267
+Nconf = 0000
+csw = 1.690000
 format = 0
 BCangleT = 1.
 BCangleX = 0
 BCangleY = 0
 BCangleZ = 0
 
-# spinprojectionfname_prefix : Directory containing the loop
-# filename_prefix: name of the loop file before confindex
-# filename_prefix2: name of the loop file after the confindex
 # prefix of source and propagator fiels,
 # in case  of source / propagator io
-spinprojectionfname_prefix = /p/scratch/chbn28/hbn28d/loop/cA211a.030.32
-filename_prefix = MG_loop_lightquark_conf_conf
-filename_prefix2 = runtype_probD8_part1_stoch_NeV0
+loopextractinpath = /p/scratch/chbn28/hbn28d/loop/cB211a.25.48
+loopextractfilenameprefix = MG_loop_lightquark_conf_conf
+loopextractfilenamesuffix = runtype_probD8_part1_stoch_NeV0
+
+# The name of the loopfile contains nstoch
+loopextractnstochaccumulated = no
+
+# We do not use spin projection
+loopextractspinproject = no
+
+# We do not take the trace in spin space
+loopextractspintrace = no
+
+# The hdf5 tag in confnumber is correct (not 0004)
+loopextractlegacytraj = no
+
+# We want to filter momenta up to q^2=4
+loopextractoutqsq = 4
+
+# The original loop contains momenta upto q^2=12
+loopextractinqsq = 12
+
+# We have the loops for 64 random sources
+loopextractsamples = 64
+
+# We want also ascii files to be produced
+loopextractverbositylevel = 1
```

Comment:
can we please call this as discussed? LoopExtract_ASCII_Output

Reply:
Also, in the example input file, can you use camel-case, just to make it easier to read...